frontend/src/components/User.vue

```diff
@@ -27,6 +27,10 @@
 </template>
 
 <script>
+    import PublicUsersService from "../Core/service/PublicUsersService";
+    import HttpRequest from "../Core/HttpRequest";
+
+    const publicUsersService = new PublicUsersService(new HttpRequest('http://localhost:8080'))
```

Comment:
Tu chyba powinieneś wczytać url z pliku konfiguracyjnego zamiast tak "magicznie"
Jeśli przerzucisz bazowy url do pliku .env to by wyglądało tak
```new HttpRequest(process.env.BASE_URL)```